<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QDialog" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
                            stop:0 #e3f2fd, stop:1 #bbdefb);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">

QLabel {
background-color:#5D6D7E;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 12px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>User Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2_loginuseremail">
        <property name="styleSheet">
         <string notr="true">
QLabel {
background-color: #3498DB;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 10px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>CashierEmail:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3_loginuserpassword">
        <property name="styleSheet">
         <string notr="true">
QLabel {
background-color: #3498DB;  /* Medium gray */
color: white;

    border-radius: 5px;
    padding: 5px 10px;
font-size: 10px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #ECF0F1;
    color: #2C3E50;
    border: 1px solid #BDC3C7;
    border-radius: 3px;
    padding: 2px 4px;
    font-size: 12px;
    min-height: 20px; /* reduce height */
}

QLineEdit:focus {
    border: 1px solid #3498DB;
    background-color: #FFFFFF;
}

QLineEdit::placeholder {
    color: #95A5A6;
    font-style: italic;
}
</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_signinback_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #7F8C8D;  /* Medium gray */
color: white;
 font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
background-color: #626567;  /* Darker gray */

}
</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
 background-color: #27AE60;  /* Emerald green */
    background-color: #2ECC71;  /* Light green */
    color: white;
 font-weight: bold;
    border-radius: 5px;
    padding: 5px 10px;
}

QPushButton:hover {
background-color: #1E8449;  /* Darker shade of emerald green */

}
</string>
        </property>
        <property name="text">
         <string>Verify</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditFind"/>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
